code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -61,7 +63,8 @@
 
 /mob/living/simple_animal/bot/cleanbot/turn_on()
 	..()
-	icon_state = "cleanbot[on]"
+	icon_state = "cleanbot1[area_locked ? "r" : null]"
+
 
 /mob/living/simple_animal/bot/cleanbot/turn_off()
 	..()
```

Comment:
About the line below, you could change it to "cleanbot0" following the logic you used on turn_on()